src/_pytest/python_api.py

```diff
@@ -843,6 +843,13 @@ def raises(  # noqa: F811
         >>> with pytest.raises(ValueError, match=r'must be \d+$'):
         ...     raise ValueError("value must be 42")
 
+    The ``match`` argument searches the formatted exception string, which includes any ``__notes__``:
```

Comment:
Perhaps it would be nice to add a link to the relevant PEP?

Reply:
Added!